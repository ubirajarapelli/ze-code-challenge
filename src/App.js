import React, { useState } from 'react';
import { useQuery } from '@apollo/client';
import { AddressSuggestion } from './components/AddressSuggestion'
import { CATEGORIES } from './services'

function App() {
  const [fieldValue, setFieldValue] = useState()

  const { loading, error, data } = useQuery(CATEGORIES);
  
  console.log(loading, error, data)

  return (
    <div>
      <AddressSuggestion
        label="cityOfBirth.label"
        name="cityOfBirth"
        id="cityOfBirth"
        value={fieldValue}
        placeholder="Escolha a cidade"
        handleChange={(city) => {
          setFieldValue(city);
        }}
      />
    </div>
  );
}
export default App;
