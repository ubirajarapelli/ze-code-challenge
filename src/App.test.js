import React from 'react';
import { render }from '@testing-library/react'
import App from './App'

describe('Test from App Component', () => {
  it('should mount App Component', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });

});