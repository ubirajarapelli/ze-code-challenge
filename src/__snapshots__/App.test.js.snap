// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test from App Component should mount App Component 1`] = `
<div>
  <div>
    Hello World Echp!
  </div>
</div>
`;
